pkg/core/state/account.go

```diff
@@ -35,11 +35,12 @@ type Account struct {
 	Votes      []*keys.PublicKey
 	Balances   map[util.Uint256][]UnspentBalance
 	Unclaimed  []UnclaimedBalance
-	// NEP5Balances is a map of the NEP5 contract hashes
-	// to the corresponding structures.
-	NEP5Balances map[util.Uint160]*NEP5Tracker
 }
 
+// NEP5Balances is a map of the NEP5 contract hashes
+// to the corresponding structures.
+type NEP5Balances map[util.Uint160]*NEP5Tracker
```

Comment:
I think we can strip the pointer off here, it makes no sense, this structure is just 12 bytes.